puppet-blacksmith.gemspec

```diff
@@ -8,15 +8,15 @@ Gem::Specification.new do |s|
   s.version = Blacksmith::VERSION
 
   s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
-  s.authors = ["MaestroDev"]
+  s.authors = ["MaestroDev", "Vox Pupuli"]
   s.date = Time.now.getutc.strftime "%Y-%m-%d"
   s.summary = "Tasks to manage Puppet module builds"
   s.description = "Puppet module tools for development and Puppet Forge management"
-  s.email = ["info@maestrodev.com"]
+  s.email = ["voxpupuli@groups.io"]
   s.homepage = "http://github.com/voxpupuli/puppet-blacksmith"
   s.require_paths = ["lib"]
   s.rubygems_version = "1.8.24"
```

Comment:
Is this still needed or can we just drop this?

Reply:
dropped it

Reply:
Did you forget to push?

Reply:
ups, now it's fixed